### DISCUSSION TITLE
Is it possible to parse `<table>` (and `<td>`, etc.) tag content to render Admonitions, Tabs, etc. in it?

### DISCUSSION THREAD

**Discussion by Enn83** [facelessuser/pymdown-extensions#2333] [Q&A] [answered]
Hi 😀!

I was trying to make advanced table that looks like pytest table: 
By clicking on a line (or the first cell of a line), I bring up the `<tr>` line below, (which can contains a `<div>` tag or not) that contains himself markdown content, i.e. Admonitions, Tabs, Tables and so on. But I noticed that some markdown content inside a table are not converted to HTML.

For example, I have a table like this (with all `<tag ... markdown="1">`) :

```md
<table filteridx="0" unsortable markdown>

<thead>
<tr class="header">
<th>COL1</th>
<th>COL2</th>
<th>COL3</th>
<th>COL4</th>
</tr>
</thead>

<!-- My first multi-row table row -->
<tbody markdown>
<tr onclick="toggleDisplayFor('LINE1')">
<td style="color:black; background-color:#9cf3d5">VAL A1</td>
<td>VAL A2</td>
<td>VAL A3</td>
<td>VAL A4</td>
</tr>
<tr id="LINE1" hidden markdown>
<td class="log" fullrow markdown>
=== "Tab"
    Tab content ...

!!!info "Admonition"
    Admo. content ...
</td>
</tr>
</tbody>

<!-- Another multi-row table row... -->
<tbody>
... etc ...
</tbody>

</table>
```

For comprehension (without markdown), my table look like this (when all multirows are opened/visible):

<table filteridx="0" unsortable markdown>

<thead>
<tr class="header">
<th>COL1</th>
<th>COL2</th>
<th>COL3</th>
<th>COL4</th>
</tr>
</thead>

<tbody markdown>
<tr onclick="toggleDisplayFor('LINE1')">
<td>VAL A1</td>
<td>VAL A2</td>
<td>VAL A3</td>
<td>VAL A4</td>
</tr>
<tr id="LINE1" hidden markdown>
<td class="log" colspan="4" markdown>
Line A details that contain Admonitions, etc.
</td>
</tr>
</tbody>

<tbody markdown>
<tr onclick="toggleDisplayFor('LINE2')">
<td>VAL B1</td>
<td>VAL B2</td>
<td>VAL B3</td>
<td>VAL B4</td>
</tr>
<tr id="LINE2" hidden markdown>
<td class="log" colspan="4" markdown>
Line B details that contain Admonitions, etc.
</td>
</tr>
</tbody>

</table>

My problem is that Admonitions, Tabs, etc. are not rendered inside a table.
Is there a way to display Admonitions, Tabs, etc. inside a table `<td>` cell? 

Thanks in advance 😊!

-----

***Note:** More precisely, I include the content of another file with the `--8<--` snippet notation inside the big `<td>`. The snippet notation work inside a table, it include well the content of the specified file, even inside a table cell, but the content is not parsed from markdown :/*

***EDIT 1:** Without writing plain HTML for them, it can become hard for Tabs for example.*

**Comment by facelessuser** [comment, marked as answer]
I finally got around to looking at this. `<td>` are handled as inline by default, so you have to instruct them to be handled as `block` if you'd like them to handle block content. This is done via `markdown="block"`.

**Reply by Enn83** [reply]
It works! 
Thanks you for your precious help and sorry to reply so late :( 
(*It was the only day I didn't check email/github*)

Indeed when adding `markdown="block"` to the last `<td>` only that contain markdown syntax, all markdown is rendered correctly 😊 (including newlines).

```html
<table markdown>
<thead>...</thead>
...

<tbody markdown>
<tr onclick="toggleDisplayFor('LINE1')">
<td>A lot of TDs</td>
...
<td>A lot of TDs</td>
</tr>

<tr id="LINE1" hidden markdown>
<td class="casedata" fullrow markdown="block">
--8<-- "docs/.../case1.md"
</td>
</tr>
</tbody>

...
</table>
```

By the way, was it a question related to `pymdown-extensions` or to another extension (`python-markdown` for example)?
I'm always having trouble identifying which extension the problem/question relates to...
